src/tls.rs

```diff
@@ -475,11 +478,14 @@ pub(crate) enum TlsBackend {
     Default,
     #[cfg(feature = "native-tls")]
     BuiltNativeTls(native_tls_crate::TlsConnector),
-    #[cfg(feature = "__rustls")]
+    #[cfg(any(
+        feature = "__rustls_crypto_ring",
+        feature = "__rustls_crypto_aws_lc-rs"
```

Comment:
```suggestion
        feature = "__rustls_crypto_aws_lc"
```
at this point it's probably easier to add `-rs` to Cargo.toml

Reply:
actually I'd suggest changing this just to `rustls-base` since it doesn't compile with only that feature otherwise

Reply:
Either that or gate the `Debug` impl behind this same `cfg(any())`